tests/virt_autotest/libvirt_routed_virtual_network.pm

```diff
@@ -35,6 +35,13 @@ use version_utils 'is_sle';
 sub run_test {
     my ($self) = @_;
 
+    #Deploy guest system under default pool - /var/lib/libvirt as default on vm host
```

Comment:
This shouldn't be here as it applies for all the networking cases￼. otherwise this is ready to go.￼

Reply:
Great, got it. make make sense here. 
Thanks so much for your comments here.
Had removed it already.